public/wp-content/themes/ColoredCow/src/scss/style.scss

```diff
@@ -33,5 +33,26 @@ Text Domain: coloredcow
 		display: block;
 	}
 }
+body{
+	padding-top: 150px;
+	background-image: url('RSVP_IMAGES/bg.png');
+}
+h5{
```

Comment:
this is not a good way to apply font. If you are targeting all h5 elements in your website, then it's okay. But if you're targeting a specific element, use classes instead.